locales/en-US/messages.properties

```diff
@@ -16,16 +16,16 @@ donate_monthly=Donate monthly
 # Homepage
 donate_now=Donate now
 other_amount=Other amount
-one_time=One time
-monthly=Monthly
+one_time=ONE TIME
```

Comment:
Unfortunately this will work only in en-US, we need to retranslate those strings to get an uppercase version.

Since those are critical strings, my suggestion would be to keep this change for en-US **and** add new strings (e.g. `one_time_uppercase`). Then once a critical mass of translations has been reached, swap them. This way we avoid adding more strings behind conditionals

Reply:
Yep that’s correct, but I’m not sure that’s a en-US only thing. I would say it’s more of a design choice, and if so, it should be applied to all locales.

@kristinashu, do you have an opinion or preference about this?

Reply:
I think I keep forgetting the nuances here, but if it's a design choice I would do the all caps in css, the reason I ddon't is because I thought all caps meant different things to different cultures, and thus it was best to let the localizers make the call?

If I'm wrong on this then I can make them all caps in css which would be ideal.

Reply:
No, you’re perfectly right to think that, and that’s why I didn’t suggest a CSS fix. Using `text-transform: uppercase` is a bad practice for localized content https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_content_best_practices#Create_localizable_UI

Reply:
OK great, adding the one_time_uppercase string.